tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -0,0 +1,189 @@
+const axios = require('axios');
+const tokensFilesConfig = require('./configs/SynchronizeTypographyTokensConfig.json');
+const fileWriter = require('./helpers/fileWriter');
+const fileRead = require('./helpers/fileReader');
+const modifiers = require('./helpers/modifiers');
+const importerVariables = require('./helpers/typographyVariables');
+import * as dotenv from 'dotenv';
+import {
+	Dict,
+	ITokenJsonObject,
+	IResultJsonObject,
+	ConfigFileObject,
+	ITypographyToken,
+	ConfigFileBin,
+} from './helpers/structures';
+
+dotenv.config();
+
+const ImportTypographyRaw = (
+	name: string,
+	binValues: ConfigFileObject,
+	jsonTypography: Object,
+	isTheme: boolean,
+) => {
+	let resultCss: Array<string> = [];
+	let resultScss: Array<string> = [];
+	let resultJsonCss: Dict<Array<IResultJsonObject>> = {};
+
+	resultCss.push(modifiers.cssFileFirstLine(isTheme, name));
+
+	for (let key in jsonTypography) {
+		if (!importerVariables.excludedKeys.includes(key)) {
+			let value: any = jsonTypography[key];
+
+			for (let variableKey in value) {
+				let propertyName: string =
+					importerVariables.typographyPrefix +
+					importerVariables.jsonAttributeToCssProperty[key] +
+					'-' +
+					variableKey;
+				let propertyValue: string | undefined;
+
+				switch (key) {
+					case importerVariables.tokensTypographyInputAttributes.fontSize:
+					case importerVariables.tokensTypographyInputAttributes.lineHeight:
+						const valueBase: number = parseInt(value.base.value.replace(/[^0-9]/g, ''));
+						const valueSplit: Array<number> = value[variableKey].value.split('*');
+						const valueMultiplier: number = valueSplit[1];
+						propertyValue = valueMultiplier
+							? valueBase * valueMultiplier + 'px'
+							: valueBase + 'px';
+						break;
+					case importerVariables.tokensTypographyInputAttributes.letterSpacing:
+						propertyValue =
+							value[variableKey].value.replace(/[^0-9/.]/g, '') * 0.01 + '';
```

Comment:
why do we concatenate it with empty string?

Reply:
After adding typing to whole `value` i need to split this operations.